### ISSUE TITLE
Ingame stutter when hovering items while "Enable Flea quick sell" is toggled on

### ISSUE THREAD

**Issue by Zoxjib** [IhanaMies/LootValue#5]
Each item hovered-over sends the server a request and this is felt in-game as stutters. It's especially disrupting if the server is hosted remotely and feels worse with higher latency.

```
[Client Request] *** \LootValue\GetItemLowestFleaPrice
[Client Request] *** \LootValue\GetItemLowestFleaPrice
[Client Request] *** \LootValue\GetItemLowestFleaPrice
```

**Comment by mattpsvreis** [comment]
Yep! I play with friends through FIKA and one of my friends who is hosting has a 150ms latency between us, so every time I hover over a few items it stutters - more dramatically noticed with weapons which turns into like 2 to 5 seconds freezes. It makes looting a fucking drag and very dangerous in open fields seeing I'm blind and deaf for a long period of time.
